apps/homescreen/js/group.js

```diff
@@ -0,0 +1,305 @@
+'use strict';
```

Comment:
nit: I'd rather have this file called `homescreen-group.js` to indicate it's a web component.
Or maybe place it in `js/webcomponent/group.js`.
Thought?